apps/accounts/api/account/pam_dashboard.py

```diff
@@ -39,8 +39,8 @@ def get_account_risk_data(_all, query_params):
         agg_map = {
             'total_long_time_no_login_accounts': ('long_time_no_login_count', Q(risk='long_time_no_login')),
             'total_new_found_accounts': ('new_found_count', Q(risk='new_found')),
-            'total_group_changed_accounts': ('group_changed_count', Q(risk='group_changed')),
-            'total_sudo_changed_accounts': ('sudo_changed_count', Q(risk='sudo_changed')),
+            'total_groups_changed_accounts': ('groups_changed_count', Q(risk='groups_changed')),
+            'total_sudoers_changed_accounts': ('sudoers_changed_count', Q(risk='sudoers_changed')),
             'total_authorized_keys_changed_accounts': (
                 'authorized_keys_changed_count', Q(risk='authorized_keys_changed')),
             'total_password_expired_accounts': ('password_expired_count', Q(risk='password_expired')),
```

Comment:
```py

def get_account_risk_data(_all, query_params):

    # Initialize aggregation map dictionary
	agg_map = {

```

The code doesn't seem fully implemented yet. I could appreciate further clarifications on what you would like to analyze in this file or if anything is incorrect according to the latest standards. For instance, there seems to be unnecessary lines and unclear variable names within the given code snippet that need more explanation.

If no issues were found, please let me know so we can continue with other tasks or discussions!